pydoctor/themes/base/apidocs.css

```diff
@@ -528,3 +528,9 @@ div.tip, div.hint, div.important {
     background-color: #d9edf7;
     border-color: #bce8f1;
 }
+
+/* Version modified style */
```

Comment:
Not sure if this CSS comment has any value.
The `.rst-versionmodified` is quite descriptive.